tests/integration/test_charm.py

```diff
@@ -75,8 +81,101 @@ async def test_database_is_up_as_replica_set(ops_test: OpsTest, unit_id: int) ->
     client.close()
 
 
+async def test_all_units_in_same_replica_set(ops_test: OpsTest) -> None:
+    """Tests that all units in the application belong to the same replica set.
+
+    This test will be implemented by Raul as part of his onboarding task.
+
+    Hint: multiple ways to do this, here are two but there are probably
+    other ways to implement it:
+    - check that each unit has the same replica set name
+    - create a URI for all units for the expected replica set name and see if
+      you can connect
+    """
+    pass
```

Comment:
?

Reply:
Maybe remove this test right now, and I can add it after the PR has been approved?